lib/jekyll-redirect-from/redirect_page.rb

```diff
@@ -17,6 +17,8 @@ class RedirectPage < Jekyll::Page
     # from - the (URL) path, relative to the site root to redirect from
     # to   - the relative path or URL which the page should redirect to
     def self.from_paths(site, from, to)
+      from = Liquid::Template.parse(from).render("site" => site.config)
+      to = Liquid::Template.parse(to).render("site" => site.config)
```

Comment:
For the record, `site.config` is the wrong thing here for the `site` object in Liquid. Should be `site.site_payload`. Just a head's up.

Reply:
You can pass them as locals or as register values, since the rendering is confined to redirect plugin I'm just passing them as locals. Let me know if that's the incorrect way to do it.

Reply:
https://github.com/jekyll/jekyll/blob/master/lib/jekyll/renderer.rb#L70
https://github.com/Shopify/liquid/blob/master/lib/liquid/template.rb#L164